elements/pl-file-preview/pl-file-preview.py

```diff
@@ -1,22 +1,15 @@
-import base64
-
 import chevron
-import lxml.html
 import prairielearn as pl
 
 
-def prepare(element_html, data):
```

Comment:
Shouldn't we keep this `check_attribs` call to enforce there are no attributes?

Reply:
I suppose, though I don't think we really care if there are attributes or not.

Reply:
I'd prefer to enforce strict compliance here (don't allow attributes if none are permitted) as a matter of policy. Being strict now can only help us with backwards compatibility in the future.

Reply:
I added attribute checking back in fa7326a.